crates/aptos/src/node/local_testnet/mod.rs

```diff
@@ -52,13 +52,13 @@ use tracing_subscriber::fmt::MakeWriter;
 
 const TESTNET_FOLDER: &str = "testnet";
 
-/// Run a local testnet
+/// Run a localnet
 ///
-/// This local testnet will run it's own genesis and run as a single node network
+/// This localnet will run it's own genesis and run as a single node network
 /// locally. A faucet and grpc transaction stream will run alongside the node unless
 /// you specify otherwise with --no-faucet and --no-txn-stream respectively.
 #[derive(Parser)]
-pub struct RunLocalTestnet {
+pub struct RunLocalnet {
```

Comment:
This does get a little more confusing now that the folder is named testnet, and the network is called localnet.

Reply:
Yeah, unfortunately changing that would be too breaking. Most users won't have to check the testnet folder fortunately.